GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -150,10 +153,40 @@ public DialogResult PushAndShowDialogWhenFailed()
             return PushAndShowDialogWhenFailed(null);
         }
 
+
+        private bool CheckRemotePresent()
+        {
+            if (_gitRemoteController.Remotes.Count < 1)
+            {
+                if (DialogResult.Yes == MessageBox.Show(this, _configureRemote.Text, _errorCaption.Text, MessageBoxButtons.YesNo, MessageBoxIcon.Error))
+                {
+                    // launch remote management dialog
+                    ManageRemotes(null);
```

Comment:
Rename to be more meaningfull (like `OpenRemoteDialogAndRefreshLIst`) and remove comment...